development/compiling/compiling_for_ios.rst

```diff
@@ -47,9 +47,16 @@ It can be done in three steps, first compile 32 bit version, then compile 64 bit
 
 ::
 
-    $ scons p=iphone tools=no bits=32 target=release arch=arm
-    $ scons p=iphone tools=no bits=64 target=release arch=arm64
-    $ lipo -create bin/godot.iphone.opt.32 bin/godot.iphone.opt.64 -output bin/godot.iphone.opt.universal
+    $ scons p=iphone tools=no target=release arch=arm32
+    $ scons p=iphone tools=no target=release arch=arm64
+    $ lipo bin/libgodot.iphone.opt.arm32.a bin/libgodot.iphone.opt.arm64.a -output bin/godot.iphone.opt.universal.a
```

Comment:
As per the above, the binaries won't have `arm32` extension a priori (if they do, it's a bug).

Reply:
Doh! Copy and paste error.

```
$ ls bin
libgodot.iphone.debug.arm64.a     
libgodot.iphone.opt.arm64.a
godot.osx.opt.tools.64            
libgodot.iphone.debug.fat.a       
libgodot.iphone.opt.debug.arm64.a
libgodot.iphone.debug.arm.a       
libgodot.iphone.opt.arm.a         
libgodot.iphone.opt.x86_64.a
```